### ISSUE TITLE
Relocate weaveDNS executable inside router image

### ISSUE THREAD

**Issue by awh** [weaveworks/weave#841]
* Move weaveDNS executable into weave image
* Remove weaveDNS image
* Start weaveDNS by specifying distinct entrypoint

Requires changes to script, makefile, Dockerfiles, and some docs.

Depends on #840.

**Comment by rade** [comment]
This doesn't really depend on #840.

**Comment by tomwilkie** [comment]
This has been superseded by #826
